coverage/.resultset.json

```diff
@@ -1,7 +1,541 @@
 {
```

Comment:
Whoa! lots going on here - do you mind expanding on what this file is used for?

Reply:
Just realized this is probably generated from SimpleCov! maybe in our next work session we can remove these files from GitHub so they aren't tracked anymore